CONTRIBUTING.md

```diff
@@ -205,4 +205,86 @@ protected $containers = [
+  * **contactEmail** -> Email Address to Contact for help with the SDK
+  * **services** -> Array of Services. Each service contains the following:
+    *  **name** -> The name of the service
+    *  **methods** -> Array of Methods that can be used with the service
+       * **method**  ->  HTTP Method to call
+       * **path** -> Path to API without a basePath
+       * **fullPath** -> Path to API with basePath
+       * **name** -> Name of API Method
+       * **packaging** -> Used in the CLI to specify it to create code.tar.gz for the Create Tag Route
+       * **title** -> Title of API Method
+       * **description** -> Description of API Method
+       * **security** -> 
+           * **todo** 
+       * **consumes** -> Array of Content-Type headers the API Route accepts.
+       * **cookies** -> Are cookies required? Bool
+       * **type** -> 
+       * **headers** -> Array of headers for API
+       * **parameters** -> Parameters for API
+           * **all** -> Array containing all Parameters
+           * **headers** -> Array containing parameters that go in the header
+           * **path** -> Array containing parameters that go into the path of the API URL
+           * **query** -> Array containing parameters that go into the query of the API URL
+           * **body** -> Array containing parameters that go in the body
+
+              All Parameters will have a structure like so:
+              * **name** -> Name of parameter
+              * **type** -> Parameter Type
+              * **description** -> Parameter Description
+              * **required** -> Is parameter required
+              * **default** -> Parameter Defaults
+              * **example** -> Parameter Example
+              * **array**
+                * **type** -> Array Type (only used if param type is "array")
+  * **global**
+    * **headers** -> A object containing all global headers
+    * **defaultHeaders** -> A object containing all default headers
+
+* **language** -- Information on the current language SDK
+  * **name** -> Name of language
+  * **params** -> Custom langauge specific parameters
+
+* **sdk** -- Various Metadata used for packaging and categorising
+  * **namespace** -> SDK Namespace
+  * **name** -> SDK Name
+  * **description** -> SDK Desc
+  * **shortDescription** -> SDK Short Desc
+  * **version** -> SDK Version
+  * **license** -> SDK Licence
+  * **licenseContent** -> SDK Licence content
+  * **gitURL** -> GIT URL for SDK
+  * **gitRepo** -> GIT Repo for SDK
+  * **gitRepoName** -> Git Repo Name
+  * **gitUserName** -> Git username of creator
+  * **logo** -> SDK Logo
+  * **url** -> SDK URL
+  * **shareText** ->
+  * **shareURL** ->
+  * **shareVia** ->
+  * **shareTags** ->
+  * **warning** ->
+  * **gettingStarted** -> URL to SDK Getting Started
+  * **readme** -> SDK Readme (used to generate readme.md)
+  * **changelog** -> SDK Changelog (used to generate changelog.md)
+  * **examples** -> SDK Examples
```

Comment:
This can be more descriptive

The examples attribute stores the markdown content that needs to be rendered examples/README.md file of the SDKs. Not all sdks use this. But here's an example from flutter https://github.com/appwrite/sdk-for-flutter/tree/master/example

This attribute is set here
https://github.com/appwrite/appwrite/blob/master/app/tasks/sdks.php#L58

which reads the examples from this file  https://github.com/appwrite/appwrite/blob/master/docs/sdks/dart/EXAMPLES.md